metrics/prometheus.go

```diff
@@ -1832,24 +1822,21 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 		}
 	}
 
+	if c.includedMetrics.Has(container.OOMMetrics) {
+		for _, oomInfo := range c.infoProvider.GetOOMInfos() {
+			labels, values := genLabelValues(rawLabels, oomInfo.MetricLabels)
```

Comment:
rawLabels also depends on the presence of the container. Instead of using the default labels that we can't always rely on with OOM kills, it might be better to define new labels specific to the OOM metrics